.github/workflows/ui-tests.yml

```diff
@@ -107,29 +107,24 @@ jobs:
             ./fastlane/test_output/**
 
   ui-tests-swift:
-    name: UI Tests for iOS-Swift ${{matrix.device}} Simulator
+    name: UI Tests for iOS-Swift Xcode ${{matrix.xcode}}
     runs-on: ${{matrix.runs-on}}
     strategy:
       fail-fast: false
       matrix:
         include:
           - runs-on: macos-13
             xcode: "15.2"
-            device: "iPhone 14 (16.4)"
 
           - runs-on: macos-14
             xcode: "15.4"
-            device: "iPhone 15 (17.2)"
```

Comment:
`h`: This removes testing for different iOS versions (16 and 17)

Reply:
No it doesn't. I added a comment to clarify this.

Reply:
I added a comment for clarification